src/parts/_forms.css

```diff
@@ -128,4 +134,14 @@ textarea:disabled {
 
 ::placeholder {
   color: var(--form-placeholder);
+}
+
+fieldset {
```

Comment:
This should have a margin to match the other form elements. It looks a bit off on the demo page:
![image](https://user-images.githubusercontent.com/6742479/66019166-f64d5000-e496-11e9-9cd9-ef7c721dac6d.png)

You'll probably want the same margin values as the `input, button, textarea, select` selector.

Reply:
Good call, I noticed it had a left margin imposed by the user agent stylesheet, so I zeroed out the margin and added a bottom margin to match the other elements. I also updated the padding to be consistent with the other elements. Thanks! 

Question: When someone comments on your PR and you push changes to resolve the issue, is the code author supposed to hit the Resolve button, or is the commenter supposed to? Thanks for being so helpful!

Reply:
Perfect, that's exactly what I was looking for. Great work!

I think it's fair for you to press the "Resolve conversation" button when you've fixed the issue, especially if it doesn't require more review. I think intended that this was resolved once you made the requested change. If anything new came up, I'd start a new review.

Other projects might have this called out in their contributing guidelines, but we don't have one yet. 😅